projects/kit/components/input-number/test/input-number.component.spec.ts

```diff
@@ -270,6 +270,22 @@ describe('InputNumber', () => {
             expect(component.computedValue).toBe(`-12${CHAR_NO_BREAK_SPACE}345,67`);
         });
 
+        it(`The given value from the form is correctly converted to a string with 0's at start`, () => {
```

Comment:
Are there any other edge cases worth considering in the tests?

Reply:
add one more